src/translations/en.json

```
@@ -2478,7 +2478,7 @@
             "triggers": {
               "name": "Triggers",
               "header": "When",
-              "description": "This list of triggers is what starts your automation. A trigger is a specific event happening in or around your home, for example: ''When the sun sets''.",
+              "description": "A trigger is a specific event happening in or around your home, for example: ''When the sun sets''. Any trigger listed here will start your automations.",
```

```suggestion
              "description": "A trigger is a specific event happening in or around your home, for example: ''When the sun sets''. Any trigger listed here will start your automation.",
```

@mdimjasevic my bad you're right. "Automations" plural here makes no sense. 
Sorry about that.